int32 workmode
int32 ctrmode
int32 turnmode
float32[4] speed
int32 holdcar
